templates/patterns/card/card.ui_patterns.yml

```diff
@@ -35,6 +35,10 @@ card:
       preview:
         src: 'https://placeimg.com/1000/500/people'
         alt: 'Alternative text for card image'
+    date:
+      type: array
+      label: 'Date object'
+      description: 'Date for card. Optional.'
```

Comment:
I didn't remember the ticket and it took me a while to remember how to use this. It's not really an object, it's an associative array, let's put some info in here. At least let's write that it uses the same parameters of the bcl date-block template.